frontend/src/metabase/lib/formatting.js

```diff
@@ -470,6 +473,10 @@ export function formatTime(value: Value) {
   }
 }
 
+export function formatDuration(value: Value) {
+  return moment.duration(parseInt(value), "seconds").format("h:mm:ss");
```

Comment:
The problem with a hard coded format is you are assuming a duration range. In our case we have durations that last days, weeks, months, or even years.